libs/blocks/quiz/quiz.css

```diff
@@ -11,7 +13,7 @@
 }
 
 .quiz {
-  min-height: calc(100vh - 131px); /* window height minus global nav + global footer heights */
+  min-height: calc(100vh - var(--global-height-nav)); /* window height minus global nav height from styles.css */
```

Comment:
I'm seeing a permanent scrollbar with these values in chrome, however, if I add a -1px after the var, it seems to resolve it. Maybe we need to adjust this a bit. 

Also, for what it's worth, Chris Millar removed this min-height stuff in his quiz-perf branch due to some cls: 

https://github.com/adobecom/milo/blob/quiz-perf/libs/blocks/quiz/quiz.css#L12